assets/js/components/RtcPlayer.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { useEffect, useRef } from "react";
 
 type RtcPlayerProps = {
-  username: string;
+  clientName: string;
   name: string;
   presenterStreams: { [key: string]: MediaStream };
   streamsAvailable: { [key: string]: boolean };
 };
 
-const RtcPlayer = ({ username, name, presenterStreams, streamsAvailable }: RtcPlayerProps) => {
+const RtcPlayer = ({ clientName, name, presenterStreams, streamsAvailable }: RtcPlayerProps) => {
```

Comment:
As I look at that a second time I see that we use this variable only for checking whether a name is an equal client, so maybe it will be better to pass a boolean, instead of a string.